app/service/update-appeal-service.ts

```diff
@@ -310,23 +373,32 @@ export default class UpdateAppealService {
       }
     }
 
-    caseData.timeExtensions = appeal.previousAskForMoreTime || [];
+    caseData.timeExtensions = [];
+    const previousAskForMoreTimes = appeal.previousAskForMoreTime;
 
+    if (previousAskForMoreTimes) {
+      previousAskForMoreTimes.forEach(askForMoreTime => {
+        this.addCcdTimeExtension(askForMoreTime, appeal, caseData);
+      });
+    }
     const askForMoreTime = appeal.askForMoreTime;
-    if (askForMoreTime && askForMoreTime.status === 'submitted') {
+    if (askForMoreTime && askForMoreTime.reason) {
```

Comment:
Think the merge has gone wrong here as we should only be saving the ask for more time to CCD when it is submitted